acts_as_commentable_with_threading.gemspec

```diff
@@ -1,20 +1,20 @@
 Gem::Specification.new do |s|
   s.name     = 'acts_as_commentable_with_threading'
-  s.version  = '1.2.0'
-  s.date     = '2014-01-10'
-  s.summary  = 'Polymorphic comments Rails gem - Rails 3+ only'
-  s.email    = 'evan@tripledogdare.net'
+  s.version  = '1.3.0'
```

Comment:
Bumped the version in case we need a reference to hook onto in downstream Gemfiles.